5.1.i_have_no_vinger.py

```diff
@@ -0,0 +1,10 @@
+import datetime
+import random
+
+fDate = datetime.datetime.strptime(input("write a date (YYYY-MM-DD):"), '%Y-%m-%d')
```

Comment:
Python is snake case -
https://www.google.com/amp/s/www.theserverside.com/answer/Camel-case-vs-snake-case-Whats-the-difference%3famp=1